curriculum/challenges/english/21-a2-english-for-developers/learn-conversation-starters-in-the-break-room/657cd962b45f48127c183d7b.md

```diff
@@ -13,8 +13,9 @@ As you already studied, For most verbs conjugation can be simple. You just  add
 
 For example:
 
-`You talk` -> `He talks`
-`They do` -> `It does`
+`You talk` → `He talks`
+
+`They do` → `It does`
```

Comment:
```suggestion
Verb `to talk` conjugation:

`I talk`, `you talk`, `we talk`, and `they talk`. 
`He talks`, `she talks`, and `it talks`.

Verb `to do` conjugation:

`I do`, `you do`, `we do`, and `they do`. 
`He does`, `she does`, and `it does`.
```
This one I had to be more creative. But the principle was the same. Removing the arrows and making them more descriptive.